test/e2e/assets/chart/zz_chart.go

```diff
@@ -676,9 +676,9 @@ var _bindata = map[string]func() (*asset, error){
 	"crds/0000_50_olm_10-operatorgroup.crd.yaml":                        crds0000_50_olm_10OperatorgroupCrdYaml,
 	"crds/0000_50_olm_03-clusterserviceversion.crd.yaml":                crds0000_50_olm_03ClusterserviceversionCrdYaml,
 	"crds/0000_50_olm_04-installplan.crd.yaml":                          crds0000_50_olm_04InstallplanCrdYaml,
-	"values.yaml":                                                       valuesYaml,
-	"Chart.yaml":                                                        chartYaml,
-	".helmignore":                                                       Helmignore,
+	"values.yaml": valuesYaml,
```

Comment:
This file is automatically generated -- it looks like your editor may have decided to reformat this and then the change was accidentally committed.

Reply:
I'll exclude this and other files that are not part of the change